lib/removeSourceMap.js

```diff
@@ -0,0 +1,10 @@
+// removes embedded source maps in output css/js files:
+module.exports = (text, type) => {
```

Comment:
I tried to make this happen, but neither postcss and browserify seem to support a basic option to stop them from planting a sourcemap tag at the end of the css/js file.  Just the power to either embed the sourcemap directly or as a link to the external file.